### PR TITLE
Fix slugbuilder permissions

### PR DESCRIPTION
**Pull request by titanous**
This fixes a bug where Python deploys fail if a custom SSH client key is configured.

/cc @temujin9

### CODE REVIEW

File: test/test_git_deploy.go [hunk 1]
```diff
@@ -229,3 +230,52 @@ func (s *GitDeploySuite) TestLargeRepo(t *c.C) {
...
+	t.Assert(r.flynn("create"), Succeeds)
+
+	r.git("commit", "-m", "bump", "--allow-empty")
+	r.git("commit", "-m", "bump", "--allow-empty")
+	push := r.git("push", "flynn", "master")
+	t.Assert(push, Succeeds)
+	t.Assert(push, c.Not(OutputContains), "cached")
+
+	// cycle the receiver to clear any cache
+	t.Assert(flynn(t, "/", "-a", "gitreceive", "scale", "app=0"), Succeeds)
```
Comment (lmars) [inline test/test_git_deploy.go]: Will this not affect other tests?

Comment (titanous) [inline test/test_git_deploy.go]: Well, yeah, now that you mention it…

**Comment by titanous** [comment]
Comments addressed.

**Comment by lmars** [comment]
LGTM

**Pull request merged**